code/modules/client/preferences_gear.dm

```diff
@@ -404,7 +404,7 @@ var/global/list/gear_datums = list()
 	cost = 1
 
 /datum/gear/harness
-	display_name = "gear harness"
+	display_name = "gear harness (FBP, Dionaea)"
```

Comment:
Should change it to 'full prosthetic' since FBP is not actually an official acronym or used anywhere in the codebase.